ci/scripts/utils/launch_java_agent.sh

```diff
@@ -56,22 +92,49 @@ cd "${JENKINS_AGENT_LANUCH_DIR}"
 
 if ! [[ -f agent.jar ]]; then
   curl -sO "${controller_url}/jnlpJars/agent.jar"
+  echo "Updated agent.jar downloaded"
 fi
 
-JENKINS_TOKEN=$(cat jenkins_token)
+if [[ ! -f "${controller_user_auth_token}" ]]; then
+   echo "User Jenkins authetication TOKEN to the controller for using the Remote API does not exist"
+   exit 1
+fi
+
+JENKINS_TOKEN=$(cat "${controller_user_auth_token}")
+echo -e "#!/usr/bin/env python
+import json,sys
+with open(sys.argv[1], 'r') as file:
+    data = json.load(file)
+print(data[\"offline\"])
```

Comment:
In case `offline` isn't defined.
```suggestion
print(data.get('offline','True'))
```